packages/manager/src/queries/databases/databases.ts

```diff
@@ -135,6 +136,18 @@ export const useDatabaseMutation = (engine: Engine, id: number) => {
   });
 };
 
+export const usePatchDatabaseMutation = (engine: Engine, id: number) => {
+  const queryClient = useQueryClient();
+  return useMutation<void, APIError[], void>({
+    mutationFn: () => patchDatabase(engine, id),
+    onSuccess() {
+      queryClient.invalidateQueries({
+        queryKey: databaseQueries.databases.queryKey,
+      });
```

Comment:
We should invalidate both the Databases list and the specific database to ensure the cache is up to date.

```suggestion
      queryClient.invalidateQueries({
        queryKey: databaseQueries.databases.queryKey,
      });
      queryClient.invalidateQueries({
        exact: true,
        queryKey: databaseQueries.database(engine, id).queryKey,
      });
```

Reply:
Done